pwa/pages/about/index.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Subtitle, Title, Text } from '../../components/common/text';
+import Link from 'next/link';
+import { NextPage } from 'next';
+
+type AboutPropsType = {
+  type: AllowedElement;
+  text?: string;
+  elements?: string[];
+};
+
+type AllowedElement = 'title' | 'subtitle' | 'text' | 'list';
+
+function mapTextComponent({ type, text, elements }: AboutPropsType, index: number): React.ReactElement | undefined {
+  switch (type) {
+    case 'title':
+      return <Title title={text || ''} key={index} />;
+    case 'subtitle':
+      return <Subtitle text={text || ''} key={index} />;
+    case 'text':
+      return <Text text={text || ''} key={index} />;
+    case 'list':
+      return (
+        <ul className="list-disc pl-8" key={index}>
+          {elements && elements.map((element, idx) => <li key={idx}>{element}</li>)}
+        </ul>
+      );
+    default:
+      return undefined;
+  }
+}
+
+const aboutData: AboutPropsType[] = [
```

Comment:
Not convinced with the approach.

Reply:
What would be the best approach for you ?